flake.nix

```diff
@@ -5,7 +5,7 @@
       inputs.nixpkgs.follows = "tgi-nix/nixpkgs";
     };
     nix-filter.url = "github:numtide/nix-filter";
-    tgi-nix.url = "github:danieldk/tgi-nix";
+    tgi-nix.url = "github:danieldk/tgi-nix/moe-kernels-0.4.0";
```

Comment:
To be moved back on master no ?

Reply:
Yeah. Just to avoid accidentally updating the `tgi-nix` input, which would break MoE on master (since 0.4.0 is an API break).